src/models/Settings.js

```diff
@@ -76,6 +76,11 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, config) {
       'features.deviceFingerprinting': ['boolean', false, false],
       'features.hideSignOutLinkInMFA' : ['boolean', false, false],
       'features.customExpiredPassword': ['boolean', true, false],
+      'features.addButton': ['object', false, false],
```

Comment:
not sure you need this one, since you are explicitly defining the contents of it in the next few lines